### ISSUE TITLE
Electric scooter breaks on server

### ISSUE THREAD

**Issue by BasiqueEvangelist** [hibiii/Scooters#7]
The electric scooter [sets a tracked data to `null`](https://github.com/Hibiii/Scooters/blob/main/src/main/java/hibi/scooters/ElectricScooterEntity.java#L96), which breaks on serialization.
```
java.lang.NullPointerException: Cannot invoke "net.minecraft.util.math.BlockPos.asLong()" because "$$0" is null
        at net.minecraft.network.PacketByteBuf.writeBlockPos(PacketByteBuf.java:301) ~[server-intermediary.jar:?]
        at net.minecraft.entity.data.TrackedDataHandlerRegistry$4.write(TrackedDataHandlerRegistry.java:238) ~[server-intermediary.jar:?]
        at net.minecraft.entity.data.TrackedDataHandlerRegistry$4.write(TrackedDataHandlerRegistry.java:235) ~[server-intermediary.jar:?]
        at net.minecraft.entity.data.DataTracker.writeEntryToPacket(DataTracker.java:195) ~[server-intermediary.jar:?]
        at net.minecraft.entity.data.DataTracker.entriesToPacket(DataTracker.java:140) ~[server-intermediary.jar:?]
        at net.minecraft.network.packet.s2c.play.EntityTrackerUpdateS2CPacket.write(EntityTrackerUpdateS2CPacket.java:34) ~[server-intermediary.jar:?]
        at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:45) ~[server-intermediary.jar:?]
        at net.minecraft.network.PacketEncoder.encode(PacketEncoder.java:14) ~[server-intermediary.jar:?]
        at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:808) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025) ~[netty-all-4.1.68.Final.jar:?]
        at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:306) ~[netty-all-4.1.68.Final.jar:?]
        at net.minecraft.network.ClientConnection.sendInternal(ClientConnection.java:214) ~[server-intermediary.jar:?]
        at net.minecraft.network.ClientConnection.method_10761(ClientConnection.java:206) ~[server-intermediary.jar:?]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) [netty-all-4.1.68.Final.jar:?]
``

EDIT: Remapped stack trace to yarn

**Comment by hibiii** [comment]
Will be fixed with #9

**Comment by BasiqueEvangelist** [comment]
:heart:
